packages/calcite-components/src/components/input/input.e2e.ts

```diff
@@ -1055,20 +1055,12 @@ describe("calcite-input", () => {
       expect(Number(await element.getProperty("value"))).toBe(195);
     });
 
-    it("disallows typing any letter or number with shift modifier key down", async () => {
+    it("disallows typing any characters with shift modifier key down", async () => {
```

Comment:
Would `non-numeric characters` be more precise for this?

Applies to other test names.